RefApp/AppDelegate.swift

```diff
@@ -14,13 +14,22 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     var window: UIWindow?
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
-        if let image = UIImage(named: "redbee") {
+        
+        
+        /* if let image = UIImage(named: "redbee") {
```

Comment:
Lets clean up the unused code :)

Reply:
It was the previous implementation, will remove it